about.css

```diff
@@ -1,4 +1,4 @@
-@import url("https://fonts.googleapis.com/css2?family=Raleway&swap=display");
+@import url("https://fonts.googleapis.com/css2?family=Raleway&display=swap");
```

Comment:
Correct query parameter is display=swap

https://css-tricks.com/google-fonts-and-font-display/

Reply:
Is this more efficient? Having trouble understanding the article.

Reply:
Not more efficient, but the correct query parameter
Display is the key, swap is the value

Before it was just wrong lol